lib/system/nimscript.nim

```diff
@@ -206,12 +210,24 @@ proc mvFile*(`from`, to: string) {.raises: [OSError].} =
     moveFile `from`, to
     checkOsError()
 
+proc mvDir*(`from`, to: string) {.raises: [OSError].} =
+  ## Moves the dir `from` to `to`.
+  log "mvDir: " & `from` & ", " & to:
+    moveDir `from`, to
+    checkOsError()
```

Comment:
This is unrelated to your PR, but why is this ``checkOsError`` needed? Won't movedDir etc. raise OSError?

Reply:
The exceptions are caught in `template cbos()` and errorMsg is set there for `checkOsError()` to check and re-raise it.